### PR TITLE
fix: membership request out of fastify cors scope

### PR DESCRIPTION
**Pull request by Defferrard**

### CODE REVIEW

**Review by ReidyT — APPROVED** [review]
LGTM 🚀

**Comment by Defferrard** [comment]
I've created an Issue related to this PR
https://github.com/graasp/graasp/issues/1358

**Review by pyphilia — APPROVED** [review]
All good for me it works!

File: src/services/itemMembership/index.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ const plugin: FastifyPluginAsync = async (fastify) => {
   // schemas
   fastify.addSchema(common);
 
-  fastify.register(MembershipRequestAPI);
+  fastify.register(MembershipRequestAPI, { prefix: '/items/:itemId/memberships/requests' });
```
Comment (spaenleh) [inline src/services/itemMembership/index.ts:31]: As discussed offline, I think this hides the `:itemId` parameter. But we said that a bigger refactor is needed in order to have CORS at only 2 places (for the core api and for the apps API). This means we will have to move the cors instantiation up and extract the apps api from the item plugins.

**Review by spaenleh — APPROVED** [review]

**Pull request merged**